code/modules/uplink/uplink_items/job.dm

```diff
@@ -156,16 +156,17 @@
 /datum/uplink_item/role_restricted/magillitis_serum
 	name = "Magillitis Serum Autoinjector"
 	desc = "A single-use autoinjector which contains an experimental serum that causes rapid muscular growth in Hominidae. \
-			Side-affects may include hypertrichosis, violent outbursts, and an unending affinity for bananas."
+			Side-affects may include hypertrichosis, violent outbursts, and an unending affinity for bananas. \
+			Now also contains regenerative chemicals to keep users healthy as they exercise their newfound muscles."
 	item = /obj/item/reagent_containers/hypospray/medipen/magillitis
 	cost = 15
 	restricted_roles = list(JOB_GENETICIST, JOB_RESEARCH_DIRECTOR)
 
-/datum/uplink_item/role_restricted/gorillacubes
-	name = "Box of Gorilla Cubes"
-	desc = "A box with three Waffle Co. brand gorilla cubes. Eat big to get big. \
+/datum/uplink_item/role_restricted/gorillacube
+	name = "Gorilla Cube"
+	desc = "A Waffle Co. brand gorilla cube. Eat big to get big. \
 			Caution: Product may rehydrate when exposed to water."
-	item = /obj/item/storage/box/gorillacubes
+	item = /obj/item/food/monkeycube/gorilla
 	cost = 6
```

Comment:
I think it would be more fun to just increase the cost to compensate their new power than reduce it to one - one is lame

Reply:
Fair enough. What do you think sounds like a good asking price?

Reply:
12? I don't know just how strong you mad ethem